test/versioned/mcp-sdk/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "mcp-tests",
+  "targets": [{"name":"mcp","minAgentVersion":"13.0.0"}],
```

Comment:
the name of this should be the package name so `@modelcontextprotocol/sdk`, also the minAgentVersion will need updated when we release